pkg/analytics/up.go

```diff
@@ -191,6 +203,16 @@ func (u *UpMetricsMetadata) ExecDuration(duration time.Duration) {
 	u.execDuration = duration
 }
 
+func (u *UpMetricsMetadata) WaitedForAwakenDuration(duration time.Duration) {
+	u.waitedForAwaken = true
+	u.waitedForAwakenDuration = duration
```

Comment:
I added both a boolean and a duration because, on occasions, the wait could be True but the duration 0. But if this is too cumbersome, I'm happy to remove the boolean.